src/app/views/faq/faq.component.ts

```diff
@@ -0,0 +1,20 @@
+import {Component, OnInit} from '@angular/core';
+import {Faq} from '../../models/faq';
+import {FaqProxy} from '../../services/proxy/faq-proxy.service';
+
+@Component({
+  templateUrl: './faq.component.html',
+  styleUrls: ['./faq.component.scss'],
+  providers: [FaqProxy]
```

Comment:
Not needed here because `FaqProxy` is declared in the providers in app.module.
However `FaqProxy` will be used only here so I don't think it is needed as global provider in app.module. Delete it from there and leave this line as it is.